### ISSUE TITLE
Which licence is best for our package?

### ISSUE THREAD

**Issue by louiewang820** [UBC-MDS/socceranalysis_python#56]

**Comment by louiewang820** [comment]
Our soccer data analysis package is available for anyone to use and make changes to as they see fit. The MIT License, known for its permissive nature and minimal restrictions, is the perfect fit for our package. This allows them to freely spread the software around, as long as they keep our original copyright and license information intact. By choosing the MIT License, we can encourage more people to take advantage of our package.

**Comment by manvirsingh96** [comment]
I agree with @louiewang820 that we should use the MIT license.  We want people to be able to use our package without any hassle. MIT license is short and straightforward, which makes it easy for developers to understand and comply with. We want our package to be used and distributed both individually and commercially, which is permitted under this license. Also it disclaims warranty which means that th user is using the package at thir own risk and so we do not have to take any liability

**Comment by florawendy19** [comment]
I do agree with both your points but I think considering The Apache License  may be good. What I like about the Apache license is that it does not only give developers patent rights for derivatives of your project's code but it also does not limit how they may distribute such creations however,  the MIT license does not explicitly grant patent rights nor does it establish any ground rules for how our names for example should be treated by those who make use of our codebase.

**Comment by florawendy19** [comment]
Oh , Actually after attending the lecture on license on Wednesday , I have realized that one key point of choosing a license is how known that license is for everyone. If the license is well known like the MIT license , then people already know it and they know what is allowed and what is not allowed so it is very convenient for them to go ahead and make contributions based on the MIT license. However, if we use the license that I mentioned above, maybe for some people it is the first time they are hearing about it and they will need to go and search to find the does and don't for that license. So I do support using MIT license now.

**Comment by vincentho32** [comment]
I also think MIT  license is the best for our package. Not only the short and easy to understand structure mentioned by @manvirsingh96  ,  its popularity mentioned by @florawendy19 , and its permissive nature mentioned by @louiewang820, I also think MIT license is also the most compatible for combining code in other open source libraries. This is suitable for our package because it welcomes others to develop together which will need to use other open source libraries.

**Comment by louiewang820** [comment]
Great points! let's stick with MIT licences.
